arm-network/2017-03-01/swagger/applicationGateway.json

```diff
@@ -0,0 +1,1331 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2016-12-01"
```

Comment:
make sure that the version property in every swagger matches the api-version in file path.